doc/tips_and_tricks.md

```diff
@@ -0,0 +1,79 @@
+Tips and Tricks
+===============
+
+Setting up a virtual Python environment for building dlite
+----------------------------------------------------------
+With [virtualenvwrapper], do
+
+    pip install virtualenvwrapper
+    mkvirtualenv dlite
+    pip install -U pip
+    pip install -r requirements.txt
+    pip install -r requirements_dev.txt
+    pip install -r requirements_doc.txt
+
+    VIRTUAL_ENV=/path/to/virtual/env
+    source $VIRTUAL_ENV/bin/activate
+    mkdir build
+    cd build
+    cmake -DCMAKE_INSTALL_PREFIX=$VIRTUAL_ENV ..
+    make
+    make install
```

Comment:
```suggestion
```shell
pip install virtualenvwrapper
mkvirtualenv dlite
pip install -U pip
pip install -r requirements.txt
pip install -r requirements_dev.txt
pip install -r requirements_doc.txt

# VIRTUAL_ENV=/path/to/virtual/env
source $VIRTUAL_ENV/bin/activate
mkdir build
cd build
cmake -DCMAKE_INSTALL_PREFIX=$VIRTUAL_ENV ..
make
make install
```
```

Reply:
VIRTUAL_ENV skulle gerne allerede være automatisk sat når man er i sit virtuelle miljø. Når du installerer `virtualenvwrapper` kan du i stedet for `VIRTUAL_ENV` og `source` linjerne bruge: `workon dlite`.
Og i stedet for `mkvirtualenv` kan du bruge `mkproject`.

Reply:
> VIRTUAL_ENV skulle gerne allerede være automatisk sat når man er i sit virtuelle miljø. Når du installerer `virtualenvwrapper` kan du i stedet for `VIRTUAL_ENV` og `source` linjerne bruge: `workon dlite`. Og i stedet for `mkvirtualenv` kan du bruge `mkproject`.

Ja, det har du ret i. Har opdateret instruktionen. 

Hvad er fordelen med at bruke `mkproject`? Vi vil jo ikke lave et nyt projekt, kun et nyt virtuelt miljø.

Reply:
Fordelen er mere automatisering. Det er det samme, bare at den laver en ny mappe udenfor alle de "rå" virtualenv filer/mapper. En "arbejdsmappe". Det er bare dét. Og så CD'er den også automatisk ind i den mappe.